.env_example

```diff
@@ -1 +1,2 @@
 MYSTERIUM_API_URL=<url>
+NATS_SERVER_IP=127.0.0.1
```

Comment:
Why do we still need nats server ip at compile time instead of passing it at least on runtime command args?

Reply:
Currently compile args is the only injection engine.
Not sure if passing args is what we want, as we need some defaults.